scripts/install/auto-install.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+# auto-install.sh installs git if it isn't installed, then clones POCS and
+# installs its dependencies.
+#
+# To fetch this from github and execute it immediately, run one of these two
+# commands in a bash shell, the first if you have wget installed, the second
+# if you have curl installed:
+#
+# 1a) wget -O - https://raw.githubusercontent.com/panoptes/POCS/develop/scripts/install/auto-install.sh | bash
+# 1b) bash <(wget -O - https://raw.githubusercontent.com/panoptes/POCS/develop/scripts/install/auto-install.sh)
+#
+# 2) bash <(curl -s https://raw.githubusercontent.com/panoptes/POCS/develop/scripts/install/auto-install.sh)
+#
+# To determine whether you have these commands install, type these commands:
+#
+#   which wget
+#   which curl
+
+################################################################################
+# Env vars used for debugging of this script; these allow you to point to your
+# fork of POCS on github, so that you can download your fork instead of the
+# primary repo. There is no support for doing the same with PAWS.
+
+[[ -z "${POCS_GITHUB_USER}" ]] && export POCS_GITHUB_USER="panoptes"
+[[ -z "${POCS_GIT_URL}" ]] && export POCS_GIT_URL="https://github.com/${POCS_GITHUB_USER}/POCS.git"
+[[ -z "${POCS_BRANCH}" ]] && export POCS_BRANCH="develop"
```

Comment:
I think this is fine, but I have been thinking lately that we should go back to pushing release versions to `master` and having non-developer units always pull from `master`. Mostly the onus here would be on me to actually be good about making releases.

Reply:
I think that might be a good idea to use master that way. I.e. on rare occasions, and after testing on multiple units, we declare a new version that non-developers should use.